chips/nrf52/src/ieee802154_radio.rs

```diff
@@ -0,0 +1,1201 @@
+//! Radio driver, Bluetooth Low Energy, NRF52
+//!
+//! The generic radio configuration i.e., not specific to Bluetooth are functions and similar which
+//! do not start with `ble`. Moreover, Bluetooth Low Energy specific radio configuration
+//! starts with `ble`
+//!
+//! For more readability the Bluetooth specific configuration may be moved to separate trait
+//!
+//! ### Author
+//! * Niklas Adolfsson <niklasadolfsson1@gmail.com>
```

Comment:
remove or replace with `yourself (Brad) and Daniel Cellucci`

Reply:
Or just remove, some I think we generally want to avoid author statements for each file

Reply:
Yepp, those are annoying to maintain. I think it is sufficient with the commit history

Reply:
Turns out that whole comment was bad.